imap_processing/cdf/tests/test_imap_cdf_manager.py

```diff
@@ -0,0 +1,49 @@
+# from pathlib import Path
+# from imap_processing.cdf.cdf_attribute_manager import CdfAttributeManager
+# This is a test
+from imap_processing.cdf.imap_cdf_manager import ImapCdfAttributes
+
+# test add_instrument_global_attrs(self, instrument: str):
+
+
```

Comment:
Nice, I like this file, but don't add much more than this to it. ImapCdfAttributes is very minimal, and you don't want to duplicate tests. Really, the only test needed is that `self.source_dir` is set correctly, because right now that's the only thing done in ImapCdfAttributes, with everything else needing to be tested in `test_cdf_attribute_manager`